client/components/BudgetGrid/EntryFormRow.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component, PropTypes } from 'react';
+
+class EntryFormRow extends Component {
+  static propTypes = {
+    categories: PropTypes.object.isRequired
+  }
+
+  state = {
+    categoryId: 16,
```

Comment:
Maybe not for this PR but, once we get selectors rolling, we should have a selector for getDefaultCategory